docs/function/handlers.md

```diff
@@ -224,6 +224,21 @@ Note that path parameters (e.g. `{id}` in the example above) are available as re
 $id = $request->getAttribute('id');
 ```
 
+### Event and context
+The HTTP request event and context are available as attributes on the request:
+```php
+// Instance of Bref\Event\Http\HttpRequestEvent
+$event = $request->getAttribute('lambda-event'); 
+
+// Instance of Bref\Context\Context
+$context = $request->getAttribute('lambda-context');
+```
+
+If you're looking for the request context array, for example when using a [Lambda authorizer](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-lambda-authorizer.html#http-api-lambda-authorizer.payload-format-response):
```

Comment:
In the Symfony docs there is a rule to not use "you" or "we". Maybe rewrite this paragraph to something like: 

> The request context array is accessible in a similar way. The context array is useful with the Lambda authorizer.

Reply:
Thanks for the feedback. The Bref documentation does use the second person "you" throughout (e.g. [here](https://bref.sh/docs/function/handlers.html#autoloading) and [here](https://bref.sh/docs/function/local-development.html)) so I think it's in keeping with the style.

Reply:
Yes, let's keep it in line with Bref practices instead of Symfony practices.